docs/ops/arithmetic/Add_1.md

```diff
@@ -12,29 +12,29 @@
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
+    * *none* - no auto-broadcasting is allowed, all input shapes must match
     * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
 **Inputs**
 
-* **1**: A tensor of type T. Required.
-* **2**: A tensor of type T. Required.
+* **1**: A tensor of type T with any shape and rank. Required.
+* **2**: A tensor of type T with any shape and rank. Required.
```

Comment:
Maybe better to write _A tensor of type T and **arbitrary** rank_ ?

Reply:
Yes, it better describes intention.